packages/3d-web-client-core/src/rendering/composer.ts

```diff
@@ -91,11 +91,17 @@ export class Composer {
   private readonly gaussGrainPass: ShaderPass;
 
   private ambientLight: AmbientLight | null = null;
+  private environmentConfiguration: EnvironmentConfiguration;
```

Comment:
We can exclude this and receive+declare as a private optional property in the constructor arguments.